contribute/style-guides/documentation-style-guide.md

```diff
@@ -81,7 +81,18 @@ Always give the reader some idea of what to expect in the reference. Avoid blind
 When possible, use the exact title of the page or section you are linking to as the link text.
 
 **Example**
-* Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+Refer to the [Documentation style guide](documentation-style-guide.md) for information about word usage and capitalization guidelines.
+
+### Commands and codes
```

Comment:
what is "codes"?  

Maybe "Command line examples"?

Reply:
Ooh, much better! I was to convey something like "stuff in code blocks," but I wanted to be more specific than that. Applied your change.